modules/Bio/EnsEMBL/Variation/Pipeline/Remapping/Remapping_conf.pm

```diff
@@ -83,7 +84,7 @@ sub default_options {
             -port   => $self->o('hive_db_port'),
             -user   => $self->o('hive_db_user'),
             -pass   => $self->o('hive_db_password'),            
-            -dbname => $ENV{'USER'} . '_' . $self->o('pipeline_name'),
+            -dbname => $self->o('hive_db_name'),
```

Comment:
Shouldn't it be the same as the other pipelines? 
`-dbname => $ENV{'USER'} . '_ehive_remapping_' . $self->o('ensembl_release') . '_' . $self->o('assembly') . '_' . $self->o('species'),`